Lombiq.Tests.UI/Extensions/OrchardCoreDashboardUITestContextExtensions.cs

```diff
@@ -113,7 +111,4 @@ public static async Task GoToUsersAsync(this UITestContext context)
         await context.ClickReliablyOnAsync(By.CssSelector("#security .title"));
         await context.ClickReliablyOnAsync(By.CssSelector(".item-label.users .title"));
     }
-
-    public static Task GoToContentItemEditorByIdAsync(this UITestContext context, string contentItemId) =>
```

Comment:
This is good here, I previously asked about moving it to this class. Delete the other duplicate.